src/scene/layer-composition.js

```diff
@@ -24,9 +25,11 @@ import { RenderAction } from './render-action.js';
  */
 // Composition can hold only 2 sublayers of each layer
 class LayerComposition extends EventHandler {
-    constructor() {
+    constructor(name = "Untitled") {
```

Comment:
So this is the default name for `GraphNode`s and `Material`s too. But other classes like `Texture` default to `null`. I kind of which I'd chosen `null` or empty string for `GraphNode` and `Material` TBH. Or at least enforced a case - probably all lower case. Kinda too late to rename the default for `GraphNode` and `Material` now though.... 😢